homeassistant/components/enocean/config_flow.py

```diff
@@ -7,8 +7,7 @@
 from homeassistant.config_entries import CONN_CLASS_ASSUMED
 from homeassistant.const import CONF_DEVICE
 
-from . import dongle
-from .const import DOMAIN, ERROR_INVALID_DONGLE_PATH, LOGGER
+from . import const, dongle
```

Comment:
We prefer to import the individual names instead of whole modules.

Reply:
Sadely, with the previous version, pylint was issuing an error "
`W0611: Unused DOMAIN imported from const`
But as you can see, DOMAIN is used on line 22. I did this change as a workaround.
I will try to find another workaround, that is not incompatible with the coding standards.

Reply:
That's a bug in pylint. We don't work around it, we just disable the warning locally.

Reply:
If you use our scaffold template, you can see we have this pattern already.

https://github.com/home-assistant/core/blob/08f2714e57caa017451cb4d555bfc29b8bb088f7/script/scaffold/templates/config_flow/integration/config_flow.py#L8

Reply:
Ok, exception added.
I would not have dared to add a pylint exception on my own :-D.

Reply:
I'm working on a commit to also import individualy detect and validate_path from dongle, but for the moment, some tests are failing after the change. Manual tests shows that the flows are still working as expected though. Investigating.